docker-java/src/test/java/com/github/dockerjava/cmd/BuildImageCmdIT.java

```diff
@@ -313,6 +313,21 @@ public void cacheFrom() throws Exception {
 
     }
 
+    @Test
+    public void quiet() {
```

Comment:
this test does not seem to test much, could you please verify `getImageId` and `isBuildSuccessIndicated()`?

Reply:
Sure, will do.

Reply:
@bsideup I've opened #1318 wherein I've rebased on master and added additional checks. Please review if possible. Please note that `isBuildSuccessIndicated()` is called internally when getting image id.